src/controllers/SearchController.js

```diff
@@ -0,0 +1,185 @@
+import { Op } from 'sequelize';
+import models from '../db/models';
+import Response from '../helpers/response';
+
+const {
+  Category,
+  Art,
+  User,
+  Profile
+} = models;
+
+/**
+ * Search Controller
+ */
+class SearchController {
+  /**
+   * @static
+   * @desc POST /api/v1/search/categories/:categoryId
+   * @param {object} req
+   * @param {object} res
+   * @memberof SearchController
+   * @returns {object} arts matching the specified category
+   */
+  static async searchByCategory(req, res) {
```

Comment:
This is searching by the category id. Searching is mostly done by name or title

Reply:
Yes, so the idea is on the frontend the users get to see the category URL but on the backend, the id is being queried.

Reply:
Then the function should be named differently. Something like `getArtsByCategory`.